apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -11,6 +11,15 @@ import {
   isRosterDialogOpen
 } from './teacherSectionsRedux';
 
+const orangeButtonStyle = {
```

Comment:
nit; would it work to call this just "buttonStyle"? I don't love including implementation details like color in the name

Reply:
I went with `ctaButtonStyle` which differentiates this button's role from other buttons on the page without actually saying anything about its style properties.